.github/workflows/functional_verified.yml

```diff
@@ -0,0 +1,218 @@
+name: Functional_Verified
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "go.mod"
+      - "**.go"
+      - "**.yml"
+      - "**.yaml"
+      - "Makefile"
+      - "!deploy/kicbase/**"
+      - "!deploy/iso/**"
+    types:
+      - labeled
+      - opened
+      - synchronize
+  label:
+    types:
+      - created
+      - edited
+      - deleted
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: '1.17.1'
+
+jobs:
+  # Runs before all other jobs
+  # builds the minikube binaries
+  build_minikube:
+    if: contains(github.event.pull_request.labels.*.name, 'ok-to-test')
+    runs-on: ubuntu-18.04
```

Comment:
Not sure if we need `18.04` specifically, but if not we should use `ubuntu-latest` which will auto update to LTS without needing to update manually.

Reply:
i'm happy to try that, I was just copying what already existed